src/controller/CHIPDevice.h

```diff
@@ -322,16 +328,17 @@ class DLL_EXPORT Device
     /* Node ID assigned to the CHIP device */
     NodeId mDeviceId;
 
-    /** Address used to communicate with the device. MUST be Type::kUDP
-     *  in the current implementation.
+    /** Address used to communicate with the device.
      */
-    Transport::PeerAddress mDeviceUdpAddress = Transport::PeerAddress::UDP(Inet::IPAddress::Any);
+    Transport::PeerAddress mDeviceAddress = Transport::PeerAddress::UDP(Inet::IPAddress::Any);
```

Comment:
I believe we have an invariant somewhere that requires address to always be UDP because we ask for port and/or ip address. If we added Ble Support we have to double-check the code paths for setPort and similar.